src/tests/ftest/security/cont_delete_acl.py

```diff
@@ -52,6 +51,32 @@ def setUp(self):
         for entry in cont_acl:
             self.principals_table[entry.split(":")[2]] = entry
 
+    def error_handling(self, results, err_msg):
+        """Handle errors when test fails and when command unexpectedly passes.
+
+        Args:
+            results (CmdResult): object containing stdout, stderr and
+                exit status
+
```

Comment:
Description of err_msg is missing.

Reply:
Ah.. thanks, I keep missing those details. I have another PR I'm working on, if I don't have to rebase anymore for this PR I'll add those changes in on that PR.